modules/ivc_champva/app/services/ivc_champva/pdf_filler.rb

```diff
@@ -18,10 +19,12 @@ def initialize(form_number:, form:, name: nil)
     end
 
     def generate(current_loa = nil)
-      template_form_path = "#{TEMPLATE_BASE}/#{form_number}.pdf"
-      generated_form_path = "tmp/#{name}-tmp.pdf"
-      stamped_template_path = "tmp/#{name}-stamped.pdf"
-      FileUtils.copy(template_form_path, stamped_template_path)
+      generated_form_path = Rails.root.join("tmp/#{name}-#{SecureRandom.hex}-tmp.pdf").to_s
```

Comment:
Is there any reason we couldn't do something like this here and keep things a little simpler? `Tempfile` already guarantees a unique filename (or exception)

`generated_form_path = Tempfile.new(name).path`

Reply:
Thanks for the suggestion! The reason I'm using Rails.root.join with a specific filename pattern is that we want to ensure the generated PDF path remains in the tmp folder of the project directory. Using Tempfile.new would place the file in the system's temp directory (e.g., /var/folders/...), which differs from our desired structure (tmp/...). This approach also makes it easier to locate and manage temporary files within the project scope.

Reply:
Tempfile also allows passing a directory, but I don't feel that strongly about this and don't want to hold the PR up. The only motivation is to have simpler code :-)